.github/workflows/pyopenms-wheels.yml

```diff
@@ -202,26 +182,10 @@ jobs:
         export CFLAGS="-Xpreprocessor -fopenmp $CFLAGS"
         export CXXFLAGS="-Xpreprocessor -fopenmp $CXXFLAGS"
 
-        # create symlinks like brew https://github.com/Homebrew/homebrew-core/blob/master/Formula/qt.rb#L61
```

Comment:
This might still be needed. You need to check the resulting wheel carefully.
Or you install from brew (which you probably should do anyway because only brew offers arm versions)